spatialos-sdk/examples/project-example/main.rs

```diff
@@ -45,24 +45,83 @@ fn main() {
 
 fn logic_loop(c: &mut WorkerConnection) {
     let mut counter = 0;
+    let mut entity_id: Option<EntityId> = None;
 
     loop {
         let ops = c.get_op_list(0);
 
+        // Move entity (if it exists).
+        if let Some(ref entity_id) = entity_id {
+            // Counter == 20 is one full circle.
+            let angle = (counter as f64) * (f64::consts::PI * 2.0 / 20.0);
+            c.send_component_update::<improbable::Position>(
```

Comment:
Yes it is explicitly required.

Reply:
Hmmm I wonder if that's just a failure of the compiler to infer `C` given `C::Update` 😕

Reply:
So I tried `improbable::Position::Update` (the trait alias) instead of `improbable::PositionUpdate` (the concrete type), but Rust wanted me to opt in explicitly to that trait by writing `<improbable::Position as Trait>::Update` or similar.

Reply:
This has been a known issue for a while: https://github.com/rust-lang/rust/issues/38078. Unfortunately the best we can do about it for now is to explicitly specify the trait as @davedissian has already done 😞